content/en/Platform Deep Dive/Assets/_index.md

```diff
@@ -37,6 +37,7 @@ You can create an asset in the following ways:
 
 - In the UI:
   - Navigate to the **Assets** page, and select **New Asset**.
+    - To upload assets in bulk in CSV or XLSX format, select **Bulk Assets**. Once uploaded, you can select an asset to add an image, [technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset), and [attachments](/getting-started/assets/asset-description/#attachments) to it.
```

Comment:
Minor grammar correction. 

I'm tempted to split out the list in separate sub-bullets (image, technology stack, attachments), but the section's already pretty bulleted.

```suggestion
    - To upload assets in bulk in CSV or XLSX format, select **Bulk Assets**. Once uploaded, you can select an asset to add an image, [technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset), and [attachments](/getting-started/assets/asset-description/#attachments).
```